es74x-cloud/src/main/scala/tech/beshu/ror/proxy/Boot.scala

```diff
@@ -3,30 +3,26 @@
  */
 package tech.beshu.ror.proxy
 
-import cats.data.EitherT
+import better.files.File
 import cats.effect.{ExitCode, IO, IOApp, Resource}
 import cats.implicits._
-import com.twitter.finagle.Http
-import monix.eval.Task
-import monix.execution.Scheduler.Implicits.global
-import org.apache.http.HttpHost
 import org.apache.logging.log4j.scala.Logging
-import org.elasticsearch.client.{RestClient, RestHighLevelClient}
-import org.elasticsearch.common.settings.Settings
-import org.elasticsearch.threadpool.ThreadPool
-import tech.beshu.ror.boot.StartingFailure
-import tech.beshu.ror.proxy.es.clients.RestHighLevelClientAdapter
-import tech.beshu.ror.proxy.es.{EsCode, EsRestServiceSimulator}
-import tech.beshu.ror.proxy.server.ProxyRestInterceptorService
-import tech.beshu.ror.utils.ScalaOps._
 
-object Boot extends IOApp with Logging {
+object Boot
+  extends IOApp
+    with RorProxyApp
+    with Logging {
+
+  //TODO: load from real config
+  override val config: RorProxy.Config = RorProxy.Config(
+    targetEsNode = "http://localhost:9200",
+    proxyPort = "5000",
+    esConfigFile = File(getClass.getClassLoader.getResource("elasticsearch.yml"))
+  )
 
   override def run(args: List[String]): IO[ExitCode] = {
-    for {
-      _ <- IO(EsCode.improve())
-      startingResult <- runServer
-      exitCode <- startingResult match {
+    start
```

Comment:
Actually I don't know how to move all code to RorProxyApp and make tests runnable. In tests I need `CloseHandler`  (not `ExitCode` ?) and here we have `IO.never` which is also problematic.  Do you have any idea how it can be done better?

Reply:
Nope. I thought it's easy. But ok, we should leave it for later. It's not so important at the moment